generate.sh

```diff
@@ -63,7 +68,10 @@ for VAL in $(jq "(.paths[][] | select(.security[0].evesso).security[0].evesso[0]
   fi
   echo "public static final String $UPPER = \"$VAL\";" >> $FILE
 done
-echo -e "\npublic static final String[] ALL = {$ALL};" >> $FILE
+echo -e "\nprivate static final String[] ALL_VALUES = {$ALL};" >> $FILE
+echo "" >> $FILE
+echo "    public static Set<String> ALL = new HashSet<>(Arrays.asList(ALL_VALUES));" >> $FILE
```

Comment:
Purely style, but you might consider wrapping the assignment to ALL with Collections.unmodifiableSet to reflect the constant nature.